build.gradle

```diff
@@ -249,6 +277,54 @@ task deploy() {}
 
 subprojects {
 
+  if (file('src/main/java').directory) {
+    apply plugin: 'com.jfrog.bintray'
+    apply plugin: 'maven-publish'
+
+    publishing {
+      publications {
+        mavenJava(MavenPublication) {
+          groupId "tech.pegasys.${project.group}"
```

Comment:
(a) do we really want each sob jar to be it's own maven group?  Since they are all needed in pantheon and only used in pantheon and can't be severed from pantheon (except for two cases: metrics and plugins api) wouldn't one group make more sense? Could we all put them in to one group and let the artifact name do the work?

(b) regardless of (a) should the group name include something like "internal" or "unstable" to make it clear it's not a stable API?

Reply:
Everything in `tech.pegasys.pantheon` makes sense to me.

Including 'internal' probably makes sense.  We may need to find a way to override some of this info on a per-package basis (eg would be nice if metrics wasn't marked internal) but starting with everything internal and then some migrating out if/when we work out how to do that makes sense to me.